### PR TITLE
Create HDZero_race2 vtx table

### PR DESCRIPTION
**Pull request by sugaarK**
vtx table for new 20x20 race VTX from HDzero

manual found here

[HDzero race2 manual]( https://28003cc0-88ab-4618-8fa6-9747a051df97.filesusr.com/ugd/ea052e_18c5903fbb13440b8d3831a965ea8a37.pdf?fbclid=IwAR2MdGKn8dIcT90CAUp74sYg4ycZYn87aOQpy_Fe5KYzVkfovMFpVew2sok)

### CODE REVIEW

File: presets/4.3/vtx/HDZero_race2.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#$ TITLE: TBS Unify Pro32 Nano - SA 2.1
```
Comment (limonspb) [inline presets/4.3/vtx/HDZero_race2.txt:1]: Title :)

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.3/vtx/HDZero_race2.txt (hunk 1 above)
Comment (sugaarK) [reply presets/4.3/vtx/HDZero_race2.txt]: opps yeah

**Review by limonspb — COMMENTED** [review]
According to HDZero manual, it is a SmartAudio 2.1. For SA 2.1 power values should be in db.
However, the VTX table shared in the manual it shows `powervalues 1 2 3`

I guess it's a question to the HDZero developers, is that a typo in the user manual or something else?
image

**Comment by limonspb** [comment]
seems like HDzero are about to fix something:
image

**Pull request closed without merging**